### ISSUE TITLE
Can't Ignore directory, such as frameworks !!!!!

### ISSUE THREAD

**Issue by Detailscool** [CocoaPods/CocoaPods#12262]
https://github.com/CocoaPods/CocoaPods/blob/e8857050bb70e6792ed9f50fde182ba7e8e6c08a/lib/cocoapods/downloader/cache.rb#L332

**Comment by Detailscool** [comment]
lose frameworks when podpsepec declares vendored_frameworks during copy

**Comment by puls** [comment]
I believe I'm also seeing this problem: for example, `pod install` in my repo gives me a `Pods/PLCrashReporter` folder with only its `LICENSE.txt` file and nothing else.
<img width="686" alt="Screenshot 2024-02-06 at 12 30 04 AM" src="https://github.com/CocoaPods/CocoaPods/assets/9763/de595d00-7c36-4faa-b208-664c0392b6a0">

**Comment by nikolaykasyanov** [comment]
I'm experiencing a similar issue with the following Podfile:
```ruby
platform :ios, '17.0'
inhibit_all_warnings!
use_frameworks!

target 'CocoaPodsReproducer' do
	pod 'Instabug', '12.6.0', :source => 'https://cdn.cocoapods.org/'
end
```

The error:
```
$ pod install
Analyzing dependencies
Downloading dependencies
Downloading Instabug (12.6.0)
Installing Instabug (12.6.0)
Generating Pods project
[!] The plist file at path `/Users/User/Desktop/CocoaPodsReproducer/Pods/Instabug/Instabug.xcframework/Info.plist` doesn't exist.
```

**Comment by mzaien** [comment]
> I'm experiencing a similar issue on 1.15.1 with the following Podfile, no problems on 1.15.0 though:
> 
> ```ruby
> platform :ios, '17.0'
> inhibit_all_warnings!
> use_frameworks!
> 
> target 'CocoaPodsReproducer' do
> 	pod 'Instabug', '12.6.0', :source => 'https://cdn.cocoapods.org/'
> end
> ```
> 
> The error:
> 
> ```
> $ pod install
> Analyzing dependencies
> Downloading dependencies
> Downloading Instabug (12.6.0)
> Installing Instabug (12.6.0)
> Generating Pods project
> [!] The plist file at path `/Users/User/Desktop/CocoaPodsReproducer/Pods/Instabug/Instabug.xcframework/Info.plist` doesn't exist.
> ```

Same but with Intercom, we are using RN btw if this helps

**Comment by paulb777** [comment]
Fixed in 1.15.2
